.gitignore

```diff
@@ -6,3 +6,4 @@ doc/_build/*
 doc/_static/*
 doc/_templates/*
 test-*
+.DS_Store
```

Comment:
Probably should have this as a global/personal .gitignore setting. This is mac specific and not likely to come up during Segway development since it's not one of it's targets.